gasoline/schema.py

```diff
@@ -95,6 +101,7 @@ class Meta:
         model = Station
         filter_fields = {
             'id':['exact'],
+            'station__name': ['exact'],
```

Comment:
The field 'station__name' doesn't exists. Please check

It is causing this error:

![image](https://user-images.githubusercontent.com/44986369/97511906-2b57fa80-1956-11eb-9e23-dcc760fcd61d.png)